### DISCUSSION TITLE
How to get quick fix suggesting just like i get in vscode php

### DISCUSSION THREAD

**Discussion by khanakia** [zed-industries/zed#10798] [Feature Requests]
Vscode when i type `CMD + .` it does shows the quick fix so i can import
![Screenshot 2024-04-20 at 2 09 45 PM](https://github.com/zed-industries/zed/assets/2174170/2d45735e-0894-4c43-a59c-d09a091c4cc8)

But in zed editor there is no way to get quick fix suggesting all I have to do is delete the `Media` keyword and start typing.
What if I copy paste some code from somewhere so I want to quick fix the imports.

![Screenshot 2024-04-20 at 2 10 28 PM](https://github.com/zed-industries/zed/assets/2174170/78392fb4-7817-4f35-8d98-dbabbfc88c3a)

**Comment by hermannsorgel** [comment]
Looks like it is actually possible and depends on LSP:
https://github.com/zed-industries/zed/issues/10177

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
